test/src/components/Input-test.js

```diff
@@ -22,8 +22,7 @@ describe('Input', () => {
     render(<Input iconClass='linkedin' />).findTag('i').should.be.ok;
   });
 
-  it('shows a popup on focus if data-content is given', () => {
-    let renderedInput = render(<Input dataContent='Sample Popup'/>).first();
-    renderedInput.elm.popup.called.should.equal(true);
+  it('has an icon class if iconClass given', () => {
+    render(<Input iconClass='foo' />).findClass('ui icon input');
```

Comment:
Added test for the tiny magic of adding the icon class if iconClass prop is given.  I removed the functionality by mistake but didn't realize it until doing annotations.  Now, a test will fail if it is missing/broken.